classes/report/fields/runner/event/nyancat.php

```diff
@@ -80,7 +80,8 @@ public function __toString()
 
 			$string .= $this->clear();
 
-			for ($row = 0; $row < count($cat); $row++)
+            $count = count($cat);
+			for ($row = 0; $row < $count; $row++)
```

Comment:
I know atoum uses `sizeof` everywhere but we should use `count` because the former is an alias :-/.